README.md

```diff
@@ -41,6 +41,12 @@ If you use ``conda``, you can install as:
 conda install -c conda-forge jupyterlab
 ```
 
+or you may update conda before installing jupyterlab as:
```

Comment:
Thank you for the patch! 

To help us understand this more, can you tell us what specific issues you ran into that were solved by updating conda? Updating conda without an explicit reason can cause issues in other packages that are unrelated to lab, so if we put it in a prominent place like the README we should be careful to provide concrete reasoning.

Reply:
@jzf2101 do we have a 'troubleshooting install issues' section in this repo or elsewhere?

Reply:
We should probably have that I don't believe that should probably go here: 

http://jupyterlab.readthedocs.io/en/stable/getting_started/installation.html

Reply:
@yuvipanda as I mentioned before, I ran into error to update conda first on virtualenv while installing jupyterlab and needed to update conda before installing.

Reply:
Can you help me understand what you mean by 'update conda first on virtualenv'? Are you using virtualenv to install conda?

Reply:
I have conda in virtualenv and wanted to install lab. I am able to work but while installing, I was unable to do with your installation code as I got error, your current version doesn't support the command etc, I am not very sure about error but after updating conda, I was able to.

Reply:
I don't think installing conda inside virtualenv is a common configuration, and I think it might cause more confusion with unrelated package breaking when put here.

Can you provide what error you got?

Reply:
I have written enough information already. I thought it might be helpful. If you don't want to add, thats fine.

Reply:
@HariPadBharti thanks for your contribution. A few comments:

* In the future, I think we should our documentation efforts on the main ReadTheDocs - we can have simple installation directions on the README, but they should be abbreviated and point to the main docs on ReadTheDocs.
* In this case, I think updating conda is mostly outside the scope of our documentation. In the main ReadTheDocs, we could add a troubleshooting section at the end of the install page, but I am hesitant to put such a special case thing, right in the middle of the installation all users will see.